core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v1/RecipeEndpoint.java

```diff
@@ -28,46 +28,16 @@
 public interface RecipeEndpoint {
 
     @POST
-    @Path("user")
```

Comment:
if we remove the  @Path("user") variables then we break the api completely, I would keep the @Path("user") or @Path("account") everywhere even if it looks ugly at the moment.